packages/docusaurus-theme-classic/src/theme/DocItem/index.tsx

```diff
@@ -67,38 +67,41 @@ function DocItem(props: Props): JSX.Element {
           })}>
           <DocVersionSuggestions />
           <div className={styles.docItemContainer}>
-            <article>
+            <article className="markdown">
               {showVersionBadge && (
-                <div>
-                  <span className="badge badge--secondary">
-                    Version: {version.label}
-                  </span>
-                </div>
+                <span className="badge badge--secondary">
+                  Version: {version.label}
+                </span>
+              )}
+
+              {/*
+              Title can be declared inside md content or declared through frontmatter and added manually
+              To make both cases consistent, the added title is added under the same div.markdown block
+              See https://github.com/facebook/docusaurus/pull/4882#issuecomment-853021120
+              */}
+              {shouldAddTitle && <MainHeading>{title}</MainHeading>}
+
+              <DocContent />
+
+              {(editUrl || lastUpdatedAt || lastUpdatedBy) && (
+                <footer className={clsx('row', styles.docUpdateDetails)}>
+                  <div className="col">
+                    {editUrl && <EditThisPage editUrl={editUrl} />}
+                  </div>
+
+                  <div className={clsx('col', styles.lastUpdated)}>
+                    {(lastUpdatedAt || lastUpdatedBy) && (
```

Comment:
Like the `EditThisPage` component, `LastUpdated` component should _not_ use the grid system (`col` class), but simply display the content it needs. After that, these two columns are consistent with each other.